### ISSUE TITLE
1.5.0 Documentation update

### ISSUE THREAD

**Issue by denis-yuen** [ga4gh/dockstore#1758]
Going to keep a running log here since there will be a large number of small issues

- ~Docker instructions link to Docker EE instructions rather than CE~
https://docs.docker.com/install/linux/docker-ce/ubuntu/
- ~Under "Testing Locally" sample_configs.local.json is missing a file format and thus crashes cwltool with the new cwltool version~
- https://docs.dockstore.org/docs/prereqs/getting-started-with-cwl/#describe-your-tool-in-cwl uses 1.25-6_1.1 but that doesn't exist and doesn't match the docker image built in  https://docs.dockstore.org/docs/prereqs/getting-started-with-docker/#testing-the-docker-image-locally failing the local launch command. (we actually create 1.25-6_1.1 later, doesn't flow well) When fixing this, try out the whole tutorial to check the flow of things
- ~https://docs.dockstore.org/docs/publisher-tutorials/getting-started-with-dockstore/#linking-with-external-services needs updates~
- ~Can mention beta sharing https://docs.dockstore.org/docs/publisher-tutorials/getting-started-with-dockstore/#sharing-the-tool~
  - ~should probably document how to use SAM based sharing in detail on a page linked to this~
- ~next steps should probably link to workflow tutorial with increasing importance of workflows~
- ~https://docs.dockstore.org/docs/publisher-tutorials/workflows/ should talk about difference between tools and workflows in the CWL/WDL context~

- More TBD

**Comment by denis-yuen** [comment]
* Disabled Nextflow option for hosted workflows should say coming soon as a tooltip or similar

**Comment by denis-yuen** [comment]
> https://docs.dockstore.org/faq/#how-does-launching-with-dockstore-cli-compare-with-cwltool

cwltool does JSON parameter generation now. We should draw the comparison with a polyfill, basically the Dockstore CLI should eventually shrink as Cromwell/cwltool improve 

> https://docs.dockstore.org/faq/#what-environment-do-you-test-tools-in
We could potentially update this to Ubuntu 18 LTS based on @garyluu 's results

> https://docs.dockstore.org/faq/#what-is-a-verified-tool-or-workflow
Need to expand with description of platform verification as added for the GA4GH testbed

> https://docs.dockstore.org/faq/#how-to-run-dockstore-with-python-3
Sounds awkward, maybe "How can I use the Dockstore CLI with Python 3?" and emphasize that this is only applicable when using cwltool/CWL

> https://docs.dockstore.org/faq/#how-do-i-add-other-users-as-maintainers-of-a-tool

Finally, for participants of the limited sharing beta, you can enter the email addresses of the users you wish to share with ...

**Comment by denis-yuen** [comment]
https://docs.dockstore.org/docs/#built-with-quayio-and-github

Update graphic and surrounding prose with page 12 graphic from https://docs.google.com/presentation/d/1aPA0qQ6c2TNBvO87ISPYmroi1JTAXkBSeiLB9QTUA50/edit?usp=sharing
To emphasize new options

**Comment by denis-yuen** [comment]
https://docs.dockstore.org/docs/user-tutorials/faceted-search/

Update graphics for material, mention search by file format and link to CWL tutorial on the subject, 

https://docs.dockstore.org/docs/user-tutorials/

We should add DNANexus

**Comment by denis-yuen** [comment]
https://docs.dockstore.org/docs/publisher-tutorials/docker-registries/
https://docs.dockstore.org/docs/publisher-tutorials/workflows/

Material updates

**Comment by denis-yuen** [comment]
New documentation needed for SAM-based sharing, hosted workflows, and nextflow added to https://docs.dockstore.org/docs/user-tutorials/language-support/.

**Comment by denis-yuen** [comment]
See also #1594

**Comment by denis-yuen** [comment]
https://docs.dockstore.org/docs/

Should emphasize WDL more in parallel with CWL.

**Comment by denis-yuen** [comment]
Make new issue for launch-with instructions for 1.5.x for nextflow with hosted workflows
Can recommend download zip, expand, and then run locally in launch-with instructions 
http://odl-dyuen2.res.oicr.on.ca:4000/docs/user-tutorials/language-support/

**Comment by denis-yuen** [comment]
Need review on http://odl-dyuen2.res.oicr.on.ca:4000/docs/user-tutorials/language-support/ for Nextflow to further explain CLI

**Comment by denis-yuen** [comment]
Closing in favour of more focused issues
